src/views/corporate-training/detail/fixed-programs/sections/JadwalKelas/components/IntakeList/IntakeCard.tsx

```diff
@@ -35,6 +42,38 @@ const IntakeCard: FC<IntakeCardProps> = ({
   const desktop = useMediaQuery(theme.breakpoints.down('lg'))
 
   const { days, hours } = practiceSessionSchedule
+  const { corporateSaleStartDate, corporateSaleEndDate } = intakeItem
+  const currentDate = new Date()
+  const convertCorporateSaleStartDate = new Date(corporateSaleStartDate)
+  const convertCorporateSaleEndDate = new Date(corporateSaleEndDate)
+
+  const checkRegisterRequirement = useMemo(() => {
```

Comment:
```suggestion
  const checkRegisterRequirement = useMemo(() => {
```
nama variablenya ga cocok pake "check" karena returnnya boolean. lebih cocok "is" atau yg mirip istlahnya. case disini bisa kaya misalnya "isNotAvailableForSale"

Reply:
hooo okehh mas